### PR TITLE
Update client implementations on tools page

### PR DESCRIPTION
**Pull request by defect**
Updating the docs quickly to mention the Go client library, as well as change the text around other languages slightly to reflect reality.

### CODE REVIEW

**Comment by byxorna** [comment]
🚢

**Comment by schallert** [comment]
LGTM

**Comment by roymarantz** [comment]
👍

**Comment by Primer42** [comment]
LGTM

File: _includes/tools/sdk.html [hunk 1]
```diff
@@ -15,10 +15,16 @@
   The ruby gem is available via <a target="_blank" href="https://rubygems.org/gems/collins_client">rubygems.org/gems/collins_client</a>.
 </p>
 <p>
-  In addition to the Ruby implementation we have a modest Bash library available that has support
-  for much of the basic core API endpoints. This is the library we use for Invisible Touch.
+  There is also a feature-complete API client written in Go. The source code is
+  available at <a target="_blank" href="https://github.com/tumblr/go-collins">github.com/tumblr/go-collins</a>.
+  The full documentation can be found on <a target="_blank" href="https://github.com/tumblr/go-collins">GoDoc</a>.
```
Comment (Primer42) [inline _includes/tools/sdk.html]: Should this be a link to https://godoc.org/github.com/tumblr/go-collins/collins?

**Comment by Primer42** [comment]
Actually, please see my comment about the godoc link.

File: _includes/tools/sdk.html (hunk 1 above)
Comment (defect) [inline _includes/tools/sdk.html]: Yes it should :) Updating now

**Comment by Primer42** [comment]
LGTM

**Pull request merged**